lib/taskana-spring-example/src/main/resources/customdb.properties

```diff
@@ -14,6 +14,7 @@ customdb.datasource.driverClassName=org.h2.Driver
 customdb.datasource.hibernate.dialect=org.hibernate.dialect.H2
 customdb.datasource.username=sa
 customdb.datasource.password=sa
+taskana.defaultSchemaName=TASKANA
```

Comment:
This is supposed to be a separate DB. Why do we need this property here? The tables of the custom DB should explicitly be in aonther schema than TASKANA.